scipy/interpolate/fitpack2.py

```diff
@@ -1529,7 +1529,7 @@ class RectSphereBivariateSpline(SphereBivariateSpline):
         (0, pi).
     v : array_like
         1-D array of longitude coordinates in strictly ascending order.
-        Coordinates must be given in radians, and must lie within (0, 2pi).
+        Coordinates must be given in radians, and must lie within (-pi, pi).
```

Comment:
This isn't quite right--the FITPACK documentation requires that

- ``-pi <= v[0] < pi``
- ``v[i] < v[i+1]`` (the strictly ascending bit)
- ``v[-1] <= v[0] + 2*pi``.

(Changed the indexing from the documentation for python conventions.) So for example in #6556 the example
```python
>>> u = np.linspace(1, 3)
>>> v = np.linspace(np.pi - 1e-6, 6)
>>> r = np.zeros([u.size, v.size])
>>> RectSphereBivariateSpline(u, v, r)
```
works fine even though 6 is not in the interval ``(-pi, pi)``.